packages/cli/src/tools/config/errors.ts

```diff
@@ -1,36 +1,28 @@
 import {CLIError} from '@react-native-community/cli-tools';
-import {ValidationError} from '@hapi/joi';
+import {ValidationError} from 'joi';
 
 export class JoiError extends CLIError {
   constructor(joiError: ValidationError) {
-    super(
-      joiError.details
-        .map(error => {
-          const name = error.path.join('.');
-          switch (error.type) {
-            case 'object.allowUnknown': {
-              const value = JSON.stringify(
-                error.context && error.context.value,
-              );
-              return `
-                Unknown option ${name} with value "${value}" was found.
-                This is either a typing error or a user mistake. Fixing it will remove this message.
-              `;
-            }
-            case 'object.base':
-            case 'string.base': {
-              const expectedType = error.type.replace('.base', '');
-              const actualType = typeof (error.context && error.context.value);
-              return `
-                Option ${name} must be a ${expectedType}, instead got ${actualType}
```

Comment:
Removed this, because the only difference was "instead got ..." which is not super helpful anyway.